doc/source/examples/multiphysics/laser-heating/laser-heating.rst

```diff
@@ -0,0 +1,231 @@
+==========================
+Laser heating
+==========================
+
+This example simulates a three-dimensional solid block heated with a laser beam. 
+
+----------------------------------
+Features
+----------------------------------
+- Solver: ``gls_navier_stokes_3d`` 
+- Laser heat source
+- Convection-radiation heat transfer boundary condition
+- Unsteady problem handled by an adaptive BDF1 time-stepping scheme 
+- Time-dependent laser path
+- Mesh adaptation using temperature
+
+
+------------------------
+Location of the example
+------------------------
+``examples/multiphysics/laser_heating/laser_heating.prm``
+
+
+-----------------------------
+Description of the case
```

Comment:
Agreed

Reply:
Based on your explanation later on for the laser parameters, this would be clearer if you also show "radial distance from the laser focal point and axial distance from the laser focal point" on this scheme. (At least in one position, for instance the time zero)

Reply:
Added.